### ISSUE TITLE
Validation usability: validation status is not obvious, perceivable, or announced.

### ISSUE THREAD

**Issue by jaclinec** [uswds/uswds#5642]
# Summary
Users did not notice the status of validation before or after conditions were met, as it doesn't announce when the conditions are met. The visual cue of the green checkmark is not noticeable to sighted users and the way it is announcing to screener readers is unclear and not working now. It also does not seem to match user expectations for how their inputs should be validated.

## Observations
All 5 participants (2 with some vision, 3 blind) in usability testing (3 using a screen reader, 1 using screen magnification and screen reader, and 1 using only screen magnification), did not notice the state of validation changing once they entered an accepted input. A green checkmark appears, but this had to be pointed out to all participants. Users said they expect an error message (in-line validation or error at hitting submit button) only if the input is invalid instead, rather than a confirmation at the top of the page.

## Affected user groups

- All users
- Screen magnification users
- Screen reader users

## Implementation recommendations
Consider using inline validation and error messages upon submit if conditions are not met.

## Research method
Usability testing with visually impaired participants on a high fidelity form prototype. See [research findings](https://github.com/uswds/uswds/wiki/Usability-research-findings-for-Q3-2023).

## Recommendations for next steps
We're looking into proper use cases for validation in [#2320](https://github.com/uswds/uswds/issues/5750). I think the first step is determining if and when validation should be used. From there, we can discuss the need to address this issue.

## Next steps
Once work is completed on [#2320](https://github.com/uswds/uswds/issues/5750), revisit this issue to determine next steps.

**Comment by brunerae** [comment]
Deprecation of this Component will be tracked in #6155

**Comment by brunerae** [comment]
Deprecation of this Component will be tracked in #6155
